### PR TITLE
Document GitOps RN-1.8.3

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
**NOTE**: From GitOps 1.7.0 onwards, we are documenting security issues in a new advisory format as part of errata updates.

**Aligned team**: Dev Tools

**Purpose**: To resolve the following issues:
(https://issues.redhat.com/browse/RHDEVDOCS-5216)

**OCP version this PR applies to**: enterprise-`4.10` and later

**Link to docs preview**: [Download to see the preview in your browser - Release notes for Red Hat OpenShift GitOps 1.8.3](https://drive.google.com/file/d/1kxNWkbKGIBsH7Nzn1iH-J8ovQyJFZQZk/view?usp=share_link)

**SME review**: Completed by @iam-veeramalla, @jaideepr97 
**QE review**: Completed by @varshab1210
**Peer-review**:  Completed by @rolfedh

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@iam-veeramalla, @jaideepr97, and @varshab1210: Please review this PR and provide feedback.

Please let me know if there are any more issues to be covered in this Release Notes for the 1.8.3 release. Thank you!

File: modules/gitops-release-notes-1-8-3.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-8-3.adoc:32]: @jaideepr97 Addressed in the revised PR. PTAL.

**Comment by Srivaralakshmi** [comment]
@iam-veeramalla, @jaideepr97, @harrietgrace, and @varshab1210: Here is the revised PR. 

Varsha, PTAL at the Note on `stable` channel.
Jaideep, I have addressed your suggestion PTAL.

- If there is anything more to be covered in RN 1.8.3, please let me know right away.
- If the content looks good, please provide approval. I will send the PR to peer review and keep it ready for the merge tomorrow.

File: modules/gitops-release-notes-1-8-3.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+[id="errata-updates-1-8-3_{context}"]
+== Errata updates
+
+=== RHEA-2023:112944 and RHBA-2023:112945 - {gitops-title} 1.8.3 security update advisory 
+
+Issued: 2023-04-17
+
+The list of security fixes that are included in this release is documented in the following advisories:
+
+* link:https://access.redhat.com/errata/RHEA-2023:112944[RHEA-2023:112944]
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-8-3.adoc:24]: @Srivaralakshmi I am not able to open the link from preview page

File: modules/gitops-release-notes-1-8-3.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not correctly propagated to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```
Comment (jaideepr97) [inline modules/gitops-release-notes-1-8-3.adoc:37]:
```suggestion
* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not correctly propagated to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled, and HPA parameters are correctly updated.  
link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```

Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-8-3.adoc:32]: @jaideepr97 Addressed in the revised PR. PTAL.

**Comment by Srivaralakshmi** [comment]
@iam-veeramalla, @jaideepr97, @harrietgrace, and @varshab1210: Here is the revised PR.

Varsha, as discussed, I have moved the admonition just before the matrix table - PTAL.
Jaideep, I have addressed your suggestion PTAL.

- If there is anything more to be covered in RN 1.8.3, please let me know right away.
- If the content looks good, please provide approval. I will send the PR to peer review and keep it ready for the merge tomorrow.

**Review by jaideepr97 — APPROVED** [review]
lgtm, thanks @Srivaralakshmi

File: modules/gitops-release-notes-1-8-3.adoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+== Errata updates
+
+=== RHEA-2023:112944 and RHBA-2023:112945 - {gitops-title} 1.8.3 security update advisory 
+
+Issued: 2023-04-17
+
+The list of security fixes that are included in this release is documented in the following advisories:
+
+* link:https://access.redhat.com/errata/RHEA-2023:112944[RHEA-2023:112944]
+* link:https://access.redhat.com/errata/RHBA-2023:112945[RHBA-2023:112945]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-8-3.adoc:20]: @reginapizza  and @iam-veeramalla: The Errata links are not accessible from the preview page. I think they are still not fixed? don't know if this is some permission issue?
https://access.redhat.com/errata/RHEA-2023:112944

No idea why. I need the errata links for them.

Any help here is much appreciated. TIA : )

Comment (varshab1210) [reply modules/gitops-release-notes-1-8-3.adoc:20]: @Srivaralakshmi LGTM from my side once we get appropriate links

File: modules/gitops-release-notes-1-8-3.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-8-3.adoc:19]: FYI, I am providing the following links:

- https://access.redhat.com/errata/RHBA-2023:112946
- https://access.redhat.com/errata/RHBA-2023:112945

These links go to the Customer Portal, not to the Errata Tool. The Errata Tool is internal only.
Users wouldn't even be able to access it.
Now you are seeing **404 Page not found** as they are not released yet. See their status. It says NEW FILES. They will only become visible on the CP when released, i.e. when their status is SHIPPED LIVE.
When released, the errata will have URLs like this: https://access.redhat.com/errata/RHBA-2023:112945.

File: modules/gitops-release-notes-1-8-3.adoc (hunk 4 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-8-3.adoc:20]: FYI, I am providing the following links:

- https://access.redhat.com/errata/RHBA-2023:112946
- https://access.redhat.com/errata/RHBA-2023:112945

These links go to the Customer Portal, not to the Errata Tool. The Errata Tool is internal only.
Users wouldn't even be able to access it.
Now you are seeing **404 Page not found** as they are not released yet. See their status. It says NEW FILES. They will only become visible on the CP when released, i.e. when their status is SHIPPED LIVE.
When released, the errata will have URLs like this: https://access.redhat.com/errata/RHBA-2023:112945.

**Comment by Srivaralakshmi** [comment]
@varshab1210 Please approve the PR and provide `/lgtm`.

File: modules/gitops-release-notes-1-8-3.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not propagated correctly to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled and the HPA parameters are updated correctly. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-8-3.adoc:32]: Possibly typo:
```suggestion
* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica set in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not propagated correctly to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled and the HPA parameters are updated correctly. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```

File: modules/go-compatibility-and-support-matrix.adoc [hunk 6]
```diff
@@ -14,11 +14,16 @@ In the table, features are marked with the following statuses:
 * *GA*: _General Availability_
 * *NA*: _Not Applicable_
 
+[IMPORTANT]
+====
+Before upgrading to {product-title} 4.13, move out of the `stable` channel.
```
Comment (rolfedh) [inline modules/go-compatibility-and-support-matrix.adoc:19]: The expression "move out" doesn't seem to fit here and isn't used elsewhere in the openshift-docs repo. Is there another more common way of expressing the same idea?

**Review by rolfedh — COMMENTED** [review]
Hi @Srivaralakshmi. I've offered a suggestion and a question. HTH! Nice work!

File: modules/gitops-release-notes-1-8-3.adoc (hunk 5 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-8-3.adoc:32]: @rolfedh yes, changed it to settings. Thanks!

File: modules/go-compatibility-and-support-matrix.adoc (hunk 6 above)
Comment (Srivaralakshmi) [reply modules/go-compatibility-and-support-matrix.adoc:19]: @rolfedh Addressed. Thank you!

**Comment by varshab1210** [comment]
LGTM
Thanks

**Review by rolfedh — APPROVED** [review]
Looks good to me. Nice work.

**Comment by Srivaralakshmi** [comment]
@varshab1210 Can you please approve the PR?

**Comment by varshab1210** [comment]
/lgtm
Thanks

File: modules/gitops-release-notes-1-8-3.adoc [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica settings in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not propagated correctly to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled and the HPA parameters are updated correctly. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-8-3.adoc:32]: is overwrote a correct word to use ?

**Review by iam-veeramalla — APPROVED** [review]

File: modules/gitops-release-notes-1-8-3.adoc (hunk 7 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-8-3.adoc:32]: yes.

**Review by varshab1210 — APPROVED** [review]

**Pull request merged**